File: Results/DER/Translation/translation_output/w_test/granite/codenet/java/python/p02789.py
System:
You are an expert python programmer and assistant

Question:
Translate the following code from java to python and enclose your solution inside ```python```.
A sample test case is provided below:

Test input:
3 3

Expected output:
Yes


```
import java.util.*;
public class p02789{
  public static void main(String[] args) {
    Scanner sc=new Scanner(System.in);
    int n=sc.nextInt();
    int m=sc.nextInt();
    if (n==m) {
      System.out.println("Yes");
    }
    else{
      System.out.println("No");
    }
 
  }
}
```


Answer:
```python
import sys

def main():
    n, m = map(int, sys.stdin.readline().split())
    if n == m:
        print("Yes")
    else:
        print("No")

if __name__ == "__main__":
    main()
```

Test input:
```
3 3
```

Expected output:
```
Yes
```<|endoftext|>

File: Results/DER/Translation/translation_output/w_test/granite/codenet/python/java/p02789.java
System:
You are an expert java programmer and assistant

Question:
Translate the following code from python to java and enclose your solution inside ```java```.
A sample test case is provided below:

Test input:
3 3

Expected output:
Yes


```
a = [int(i) for i in input().split()]
if a[0] == a[1]:
  print('Yes')
else:
  print('No')
```


Answer:
```java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int[] a = new int[2];
        for (int i = 0; i < 2; i++) {
            a[i] = scanner.nextInt();
        }
        if (a[0] == a[1]) {
            System.out.println("Yes");
        } else {
            System.out.println("No");
        }
    }
}
```

Test input:
```
3 3
```

Expected output:
```
Yes
```<|endoftext|>
